cypress/e2e/crypto/crypto.spec.ts

```diff
@@ -404,9 +404,9 @@ describe("Cryptography", function () {
 
             cy.get(".mx_EventTile_last")
                 .should("contain", "test encrypted from unverified")
-                .find(".mx_EventTile_e2eIcon", { timeout: 100000 })
+                .find(".mx_EventTile_e2eIcon")
```

Comment:
I set this timeout very high when I was debugging and apparently forgot to reduce it.